src/routes/AppProfile/AppSettings.tsx

```diff
@@ -57,15 +70,15 @@ const VersionListComp: React.FC<VersionListProps> = ({ activeProfile, selectedVe
                 data-state={selectedVersion === undefined ? "active" : "inactive"}
                 onClick={() => setSelectedVersion(undefined)}>
 
-                <header>Latest Release</header>
+                <header>Latest Release ({versionList.length > 0 ? versionList[0].tag : ""})</header>
                 <div>Recommended</div>
             </div>
             {
-                versionList.map(i =>
+                versionList.map((i, index) =>
                     <div
                         data-state={selectedVersion === i.uuid ? "active" : "inactive"}
                         key={i.uuid}
-                        onClick={() => setSelectedVersion(i.uuid)}>
+                        onClick={() => selectVersion(i.uuid, index)}>
```

Comment:
Async functions should be awaited within handlers.
```suggestion
                        onClick={async () => await selectVersion(i.uuid, index)}>
```